### DISCUSSION TITLE
What time zone are you in?

### DISCUSSION THREAD

**Discussion by maphew** [TriliumNext/Notes#58] [Polls]
branching from #2, for folks who are interesting in having a real time convesation, what time zones are you in? 
Polls are limited to 10 options so I've grouped by 3 and tried to spread across land masses. Add specific time zone in comments if you like.

**Comment by meichthys** [comment]
Closing this as the meeting date/time is being determined by this: https://www.when2meet.com/?25414487-FaSGo
